android/src/main/java/com/amplitude/reactnative/AmplitudeReactNativeModule.java

```diff
@@ -334,6 +336,37 @@ public void setPlan(String instanceName, ReadableMap planProperties, Promise pro
         }
     }
 
+    @ReactMethod
+    public void enableLogging(String instanceName, Boolean enableLogging, Promise promise) {
+        AmplitudeClient client = Amplitude.getInstance(instanceName);
+        synchronized (client) {
+            client.enableLogging(enableLogging);
+            promise.resolve(true);
+        }
+    }
+
+    @ReactMethod
+    public void setLogCallback(String instanceName, Callback logCallback) {
+        AmplitudeClient client = Amplitude.getInstance(instanceName);
+        class AmplitudeErrorCallback implements AmplitudeLogCallback {
+        @Override
+        public void onError(String tag, String message) {
+          logCallback.invoke(tag, message);
+          }
+        }
+        AmplitudeErrorCallback callback = new AmplitudeErrorCallback();
+        client.setLogCallback(callback);
```

Comment:
nit: anonymous inner class could make this a one liner
```suggestion
        client.setLogCallback(new AmplitudeLogCallback {
          @Override
          public void onError(String tag, String message) {
            logCallback.invoke(tag, message);
            }
         });
```

Reply:
Updated.